README.md

```diff
@@ -43,3 +43,25 @@ module.exports = {
   }
 }
 ```
+
+## Options
+
+### quote
+
+`--quote=[type]`
+
+Type of quote used in generated JS. Defaults to `single`.
+
+```
+css-to-radium style.css style.js --quote=double
```

Comment:
@alexlande -- Oh, and there isn't a `TODO` here, but this definitely needs an update ;)